src/thing_creature.c

```diff
@@ -5364,6 +5369,498 @@ void illuminate_creature(struct Thing *creatng)
     lgt->radius <<= 1;    
 }
 
+void controlled_creature_pick_thing_up(struct Thing *creatng, struct Thing *picktng)
+{
+    if (picktng->class_id == TCls_Creature)
+    {
+        set_creature_being_dragged_by(picktng, creatng);
+    }
+    else
+    {
+        if ((picktng->owner != creatng->owner) )
+        {
+            if (thing_is_workshop_crate(picktng))
+            {
+                update_workshop_object_pickup_event(creatng, picktng);
+            }
+            else if ( (thing_is_spellbook(picktng)) || (thing_is_special_box(picktng)) )
+            {
+                update_library_object_pickup_event(creatng, picktng);
```

Comment:
these are what generate the voice line we do not want I believe.

Reply:
I've removed the events, but it's just occurred to me that we may need them in multiplayer should a player take an enemy human player's stuff....

Reply:
Stealing is something else.

Reply:
But that's what the events are for?